cylc/flow/prerequisite.py

```diff
@@ -254,13 +262,22 @@ def _eval_satisfied(self) -> bool:
             ) from None
         return res
 
-    def satisfy_me(self, outputs: Iterable['Tokens']) -> 'Set[Tokens]':
+    def satisfy_me(
+        self, outputs: Iterable['Tokens'],
+        mode: Literal['skip', 'live', 'simulation', 'skip'] = 'live'
+    ) -> 'Set[Tokens]':
         """Attempt to satisfy me with given outputs.
 
         Updates cache with the result.
         Return outputs that match.
 
         """
+        satisfied_message: SatisfiedState = False
```

Comment:
Looks like the `False` value can never be used:

```suggestion
        satisfied_message: SatisfiedState
```